utils/deletefile.js

```
@@ -0,0 +1,11 @@
+const fs = require('fs');
+
+function deleteFile(fileName, callback) {
+    fs.unlink(fileName, (err) => {
+        if (err) throw err;
```

you  should return the error in the callback. that is wrong

---

done